app/src/main/java/com/keylesspalace/tusky/fragment/TimelineFragment.java

```diff
@@ -180,19 +184,21 @@ public StatusViewData apply(Either<Placeholder, Status> input) {
                 }
             });
 
-    public static TimelineFragment newInstance(Kind kind) {
+    public static TimelineFragment newInstance(Kind kind, boolean enableSwipeToRefresh) {
         TimelineFragment fragment = new TimelineFragment();
         Bundle arguments = new Bundle();
         arguments.putString(KIND_ARG, kind.name());
+        arguments.putBoolean(ARG_ENABLE_SWIPE_TO_REFRESH,enableSwipeToRefresh);
         fragment.setArguments(arguments);
         return fragment;
     }
 
-    public static TimelineFragment newInstance(Kind kind, String hashtagOrId) {
+    public static TimelineFragment newInstance(Kind kind, String hashtagOrId, boolean enableSwipeToRefresh) {
```

Comment:
maybe add more overloads for the `newInstance` methods, so `enableSwipeToRefresh` only has to be set when its actually false?

```
public static TimelineFragment newInstance(Kind kind, String hashtagOrId) {
   return  newInstance(kind, hashtagOrId, false);
}

```

Reply:
Done